README.md

```diff
@@ -1,50 +1,10 @@
+<img src="docs/logo.png" align="left" width="212px" height="192px"/>
 
-> The [Fishshell][fishshell] Framework
+> The <a href="http://fishshell.com">Fishshell</a> Framework
 
-[![Fish Version][fish-badge]][fishshell]
-[![Build Status][travis-badge]][travis-url]
-[![License][license-badge]](#LICENSE)
+[![MIT License](https://img.shields.io/badge/license-MIT-007EC7.svg?style=flat-square)](/LICENSE.md) [![Fish Shell Version](https://img.shields.io/badge/fish-v2.2.0-007EC7.svg?style=flat-square)](http://fishshell.com) [![Travis Build Status](http://img.shields.io/travis/oh-my-fish/oh-my-fish.svg?style=flat-square)](https://travis-ci.org/oh-my-fish/oh-my-fish) [![Travis Build Status](https://img.shields.io/badge/gitter-join_chat-brightgreen.svg?style=flat-square)](https://gitter.im/oh-my-fish/oh-my-fish?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-<a name="omf"></a>
-<br>
-
-<p align="center">
-  <a href="https://github.com/oh-my-fish/oh-my-fish/blob/master/README.md">
-  <img width="160px" src="https://cloud.githubusercontent.com/assets/8317250/8510172/f006f0a4-230f-11e5-98b6-5c2e3c87088f.png">
-  </a>
-</p>
-
-<br>
-
-<p align="center">
-<b><a href="#about">About</a></b>
-|
-<b><a href="#install">Install</a></b>
-|
-<b><a href="#getting-started">Getting Started</a></b>
-|
-<b><a href="#advanced">Advanced</a></b>
-|
-<b><a href="https://github.com/oh-my-fish/oh-my-fish/wiki/Screencasts">Screencasts</a></b>
-|
-<b><a href="/CONTRIBUTING.md">Contributing</a></b>
-|
-<b><a href="/FAQ.md">FAQ</a></b>
-
-  <p align="center">
-    <a href="https://gitter.im/oh-my-fish/oh-my-fish?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge">
-      <img src="https://badges.gitter.im/Join%20Chat.svg">
-    </a>
-  </p>
-</p>
-
-<br>
-
-# About
-
-Oh My Fish is an all-purpose framework for the [fishshell][Fishshell]. It looks after your configuration, themes and packages. It's lightning fast and easy to use.
-
-We love contributions, [fork and send us a PR](https://github.com/oh-my-fish/oh-my-fish/fork).
+Oh My Fish provides core infrastructure to allow you to install packages which extend or modify the look of your shell. It's  fast, extensible and easy to use.
 
 # Install
```

Comment:
I would add  `<br><br>` before `#install`.

1 `<br>` for line break in order to fix this:
![screen shot 2015-09-06 at 18 26 34](https://cloud.githubusercontent.com/assets/526122/9706326/7db206b4-54d9-11e5-844a-b5cc21fedeac.png)

And another `<br>` to give it a little bit of margin.